final_prep/mod_zero_hero.rb

```diff
@@ -55,17 +96,58 @@
 #   - citiesDestroyed (array)
 #   - luckyNumbers (array)
 #   - address (hash with following key/values: number , street , state, zip)
-
+scary_monster =
+{
+  'name' => 'Jerry',
+  'smell' => 'poorly',
+  'weight' => 50,
+  'citiesDestroyed' => ['Los Angeles', 'Kyoto', 'Needles, CA'],
+  'luckyNumbers' => [7, 8, 9],
+  'address' => {'number' => 12, 'street' => 'Monster Way', 'state' => 'Montana','zip' => 86753}
+}
 
 # Create a new class called SuperHero
 # - Your class should have the following DYNAMIC values
-#   - name 
+#   - name
 #   - super_power
-#   - age 
+#   - age
 # - Your class should have the following STATIC values
 #   - arch_nemesis, assigned to "The Syntax Error"
 #   - power_level = 100
-#   - energy_level = 50 
+#   - energy_level = 50
+
+class SuperHero
+
+  attr_accessor :name, :super_power, :age, :arch_nemesis, :power_level, :energy_level
+
+  def initialize(name, super_power, age)
+      @name = name
+      @super_power = super_power
+      @age = age
+      @arch_nemesis = "They Syntax Error"
+      @power_level = 100
+      @energy_level = 50
+  end
+
+  def say_name
+    puts @name
+  end
+
+  def maximize_energy
+    @energy_level = 1000
+    puts energy_level
+  end
+
+  def gain_power(increase)
+    @power_level += increase
+    puts power_level
```

Comment:
For these last 2 methods, your choice on if you want the new level to be printed. Most important is changing the value of the attribute (which you've done in both).